### PR TITLE
Ensure logClick is defined in the pagination component

### LINKED ISSUE #2715: Some of the next buttons on Talk not working anymore
Something broke the next buttons on Talk in a few places. In the collection see 

Recents: https://www.zooniverse.org/projects/mschwamb/comet-hunters/talk/recents - The next button doesn't work 

Talk Search: https://www.zooniverse.org/projects/mschwamb/planet-four-terrains/talk/tags/spiders

Talk threads the next button is working

### PR DESCRIPTION
**Pull request by eatyourgreens**
Fixes #2715 and #2713

### CODE REVIEW

**Comment by alexbfree** [comment]
Tested this. You can successfully change page when viewing a collection (per #2713) and talk threads (per #2715). However, I am not seeing any `change-page` events logged in Talk or any events showing details of pagination clicks in Talk. So I will merge this, and close #2713, but not close #2715 yet - i am not sure what was supposed to be being logged in Talk, will need to check.

screenshot 2016-06-23 17 25 54

screenshot 2016-06-23 17 32 49

File: app/talk/lib/paginator.cjsx [hunk 1]
```diff
@@ -31,6 +31,9 @@ module?.exports = React.createClass
 
   contextTypes:
     geordi: React.PropTypes.object
+  
+  componentDidMount: ->
+    @logClick = @context.geordi?.makeHandler 'change-page'
 
   componentWillReceiveProps: (nextProps, nextContext)->
```
Comment (alexbfree) [inline app/talk/lib/paginator.cjsx]: I wonder if we should remove this now - not sure it actually achieves anything now that the `logClick` handler is defined in `componentDidMount` (which always happens earlier)

**Comment by eatyourgreens** [comment]
I can see events being posted to Geordi when I click the next button in Talk. Here's an example, from https://www.zooniverse.org/talk/17?page=2
```json
{"time":"2016-06-29T11:59:11.193Z",
"browserTime":"2016-06-29T11:59:10.874Z",
"userID":"149",
"type":"change-page",
"relatedID":"next-page",
"projectToken":"zooTalk",
"userSeq":71,
"sessionNumber":35,
"eventNumber":5,
"userAgent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/51.0.2704.103 Safari/537.36",
"serverURL":"https://www.zooniverse.org",
"clientIP":"54.80.100.130",
"errorCode":"",
"errorDescription":"",
"id":96471}
```

**Comment by alexbfree** [comment]
That's because it's merged? Or am i missing your point....?

**Comment by eatyourgreens** [comment]
Sorry, I was replying to this
>However, I am not seeing any change-page events logged in Talk or any events showing details of pagination clicks in Talk
I thought that comment was made after the merge, but I'm not sure.

**Comment by alexbfree** [comment]
OK now i see what you meant. I've retested and I confirm i see the same. we are good.

**Pull request merged**